orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/WaitForUpInstanceHealthTask.groovy

```diff
@@ -28,10 +28,18 @@ class WaitForUpInstanceHealthTask extends AbstractWaitForInstanceHealthChangeTas
       return false
     }
 
-    if (healthProviders.any { it.state != "Up"}) {
-      return false
+    if (allUp(healthProviders) || mixOfUpAndUnknown(healthProviders)) {
```

Comment:
the original looks wrong, but here I think we want at least one UP, and the rest up or unknown. and in the case of only platform health then allow that to be considered up